### PR TITLE
Patch exporter/lease in service rather than updating

### PR DESCRIPTION
**Pull request by NickCao**

### CODE REVIEW

File: internal/service/controller_service.go [hunk 1]
```diff
@@ -105,6 +105,8 @@ func (s *ControllerService) Register(ctx context.Context, req *pb.RegisterReques
 		return nil, err
 	}
 
+	patch := client.StrategicMergeFrom(exporter.DeepCopy())
```
Comment (mangelajo) [inline internal/service/controller_service.go:108]:
```suggestion
	original := client.StrategicMergeFrom(exporter.DeepCopy())
```

Suggestion,  this way it makes more sense if you are not familiar with how "Patch(ctx, new, original)" works ?

Comment (mangelajo) [reply internal/service/controller_service.go:108]: at least I was confused at first read :)

**Comment by NickCao** [comment]
`unable to update exporter: the body of the request was in an unknown format - accepted media types include: application/json-patch+json, application/merge-patch+json, application/apply-patch+yaml"`

**Comment by NickCao** [comment]
strategic merge doesn't seem to work with exporters

**Review by mangelajo — APPROVED** [review]

**Pull request merged**